contracts/minters/open-edition-minter/src/state.rs

```diff
@@ -11,8 +11,9 @@ pub struct ConfigExtension {
     pub payment_address: Option<Addr>,
     pub nft_data: NftData,
     pub start_time: Timestamp,
-    pub end_time: Timestamp,
+    pub end_time: Option<Timestamp>,
```

Comment:
curious if this will cause errors on read

Reply:
The contract will be used for new open edition collections onwards without any migrations. It could cause indexing errors though if not accounted for.